docs/verifier/query-builder.md

```diff
@@ -2,59 +2,209 @@
+To create a query, it is essential to reference the correct schema, specifically its JSON-LD Context's URL. Users have two options: they can either enter a URL pointing to a JSON-LD Context or utilize one from the [Schema Explorer](https://schema-builder.polygonid.me). By starting to type the name (or any other keyword) of the desired schema in the Schema Explorer, a list of available schemas will appear, allowing the user to select the appropriate one.
 
 <div align="center">
-<img src={useBaseUrl("/img/query-builder-filled.png")} align="center" width="600"/>
+<img src={useBaseUrl("/img/query-builder-url.png")} align="center" width="600"/>
 </div>
-<br></br>
+<br/>
 
-The [Query Builder](https://schema-builder.polygonid.me/query-builder) outcome should look like this:
+### Schema type
+
+The JSON-LD Context of a schema in the Schema Explorer is limited to a single type. However, a custom JSON-LD Context may encompass multiple types. This particular field enables users to select the appropriate type for constructing their query. As a user-friendly feature, in cases where the JSON-LD Context is single-typed, the field is pre-filled automatically.
+
+### Attribute field
+
+Upon specifying the URL and type, the interface displays a hierarchical tree structure that showcases the attributes associated with the selected type. Queries are based on the values of these attributes, which are stored within a user's wallet as part of their credentials.
+
+Credentials consist of various primitive attributes such as strings, integers, booleans, etc. These primitive attributes may be structured in a nested format using object attributes (represented as folders in the tree). Since queries are only executable against primitive attributes, object attributes (folders), are not selectable within the tree.
 
 <div align="center">
-<img src={useBaseUrl("/img/query-builder-outcome.png")} align="center" width="600"/>
+<img src={useBaseUrl("/img/query-builder-attribute-field.png")} align="center" width="600"/>
 </div>
-<br></br>
+<br/>
+
+### Proof type
+
+This field allows specifying the verification method used by the issuer at the time of issuing the credential, which is essential for a successful query verification. While creating a query, users must carefully choose a Proof Type that aligns with the original method selected by the issuer when issuing the credential. The two options available are:
+
+- **Signature-based (SIG):** Verifies claims based on Baby JubJub (BJJ) Key Signatures. This method is straightforward, relying on the issuer's signature for authentication. It's fast and doesn't involve on-chain transactions.
+- **Merkle Tree Proof (MTP):** Verifies claims through their inclusion in a Merkle Tree, which is part of the issuer’s on-chain identity. This approach not only bolsters security but also provides a timestamp, confirming the claim's existence at a specific historical moment.
```

Comment:
In general, for v3 circuit it is possible that verifier doesn't know proof type which will be used. So `proofType` field will be missed in circuit and user will use the cred with any proof.

Reply:
At present, you have the option to select any proof type associated with V3 circuits, but not both or none (since it's radio button).

If I understand correctly, you are suggesting the addition of a third radio button option labeled "Any." This would mean that when "Any" is selected, the Circuit ID input below will exclusively display V3 circuits. Is my understanding correct?

Reply:
This would mean that when "Any" is selected, the Circuit ID input below will exclusively display V3 circuits. Is my understanding correct?  - yes.

Reply:
c.c @elias-garcia @Figmintojas

Reply:
I've created another task in Jira for this.